publisher:
  detection_image:
    topic: /maskrcnn_ros/segment_image
    queue_size: 1
    latch: true
  
  bounding_boxes:
    topic: /maskrcnn_ros/segment_boxes
    queue_size: 1
    latch: false

yolo_model:
  name: /mask_rcnn_resnet50_atrous_coco
  

